styles/style.css

```diff
@@ -0,0 +1,270 @@
+.container {
+  margin: 0 auto;
+  max-width: 1303px;
+}
+
+body {
+  font-size: 15px;
+  line-height: 20px;
+}
+
+.bg-red {
+  background-color: #f72210;
+}
+
+header {
+  color: #fff;
+}
+
+.btn-primary {
+  background-color: #fff;
+  color: #f72210;
+  width: 150px;
+}
+
+.list-bk-color {
+  background-color: #fff;
+  padding: 0;
+}
+
+.nav-item {
+  margin: 0 auto;
+}
+
+.list-bk-color li {
+  list-style: none;
+}
+
+.search-input {
+  background-image: url('https://g.newsweek.com/www/images/icon-search-glass.svg');
+  background-repeat: no-repeat;
+  background-position: 100% 50%;
+  background-size: 24px 24px;
+  height: 32px;
+  border: none;
+  width: 90px;
+}
+
+.sign-in {
+  font-size: 18px;
+  color: #fff;
+}
+
+h2 {
+  color: rgba(3, 3, 3, 0.4);
+  font: bold 16px/16px "Roboto Condensed", sans-serif;
+  text-transform: uppercase;
+}
+
+h3 {
+  font-size: 24px;
+  line-height: 24px;
+  font-family: "Roboto Condensed", sans-serif;
+  font-weight: bold;
+}
+
+h5 {
+  font-size: 17px;
+  line-height: 20px;
+}
+
+.nav-link {
+  color: black;
+  font-size: 15px;
+  line-height: 24px;
+  font-weight: bold;
+  font-family: "Roboto Condensed", sans-serif;
+}
+
+.list-item {
+  height: 134px;
+  padding: 20px 0;
+}
+
+svg {
+  fill: #fff;
+}
+
+.trapezium {
+  color: black;
+  width: 108.6px;
+  border-top: #f72210 15px solid;
+  border-left: #fff 20px solid;
+}
+
+.option-element {
+  font: bold 17px/21px "Roboto Condensed", sans-serif;
+  line-height: 20px;
+}
+
+.option-element div h5 {
+  font: bold 17px/21px "Roboto Condensed", sans-serif;
+}
+
+.option-element div p {
+  font-size: 14px;
+  line-height: 21px;
+  color: #555;
+  text-transform: uppercase;
+  margin-top: 4px;
+}
+
+.head-shot {
+  width: 63px;
+  height: 63px;
+  background-repeat: no-repeat;
+}
+
+.img1 {
+  background-image: url('https://d.newsweek.com/en/full/89510/marc-schulman.png?w=63&h=63&f=0544ec46ee21c238aad07cc8eaa35569');
+}
+
+.img2 {
+  background-image: url('https://d.newsweek.com/en/full/1575501/forman-andrew.webp?w=63&h=63&f=05c96bda48a8f56b570d9c2153136832');
+}
+
+.img3 {
+  background-image: url('https://d.newsweek.com/en/full/1575395/richard-rubenstein.webp?w=63&h=63&f=8b87758ea9d3bff05635e4e4ff170093');
+}
+
+.img4 {
+  background-image: url('https://d.newsweek.com/en/full/1506965/charlie-kirk.webp?w=63&h=63&f=dadcbab6cd325ee70d1bfe4df5a01410');
+}
+
+.debate {
+  border-top: rgb(231, 17, 60) 6px solid;
+}
+
+.blue-color {
+  border-top: rgb(107, 184, 209) 6px solid;
+}
+
+.debate img {
+  max-width: 133px;
+  height: 95px;
+}
+
+.double-line {
+  border-top: grey double 5px;
+}
+
+h6 {
+  text-transform: uppercase;
+}
+
+.double-line img {
+  width: 75px;
+  height: 75px;
+  margin-top: -20px;
+}
```

Comment:
Remember the purpose of the project is to use Bootstrap. Please favor the use of bootstrap classes in this case.